src/utils.rs

```diff
@@ -120,3 +120,16 @@ pub fn current_time() -> Duration {
         Duration::new(ts.tv_sec as u64, ts.tv_nsec as u32)
     }
 }
+
+extern crate bitflags;
```

Comment:
oh shit I missed that. extern crate should only be in the crate root. I'll fix this when I get home